docs/usb.rst

```diff
@@ -25,7 +25,23 @@ Adafruit TinyUSB Arduino Support
 Examples are provided in the Adafruit_TinyUSB_Arduino for the more
 advanced USB stack.
 
-Please note that this stack requires sketches to manually call
+To use Serial with TinyUSB, you must include the TinyUSB header in your
+sketch to avoid a compile error.
+
+.. code:: cpp
+
+    #include <Adafruit_TinyUSB.h>
+
+If you need to be compatible with the
+other USB stack, you can use an ifdef:
+
+.. code:: cpp
+
+    #ifdef USE_TINYUSB
+    #include <Adafruit_TinyUSB.h>
+    #endif
+
+Also, this stack requires sketches to manually call
```

Comment:
Does manually call Serial.begin() is required. I remember do that in the background , will check it is still the case

Reply:
The current main.cpp doesn't call `Serial.begin()` if `USE_TINYUSB` is set:

https://github.com/earlephilhower/arduino-pico/blob/a1c9dfe84d3c5070a2a7828618ebf856f84e6b9f/cores/rp2040/main.cpp#L64-L74

I should also probably put in a baud rate there, too.  D'oh.

Reply:
it is a bit complicated, but the lib is designed to be one hook called only. It will start the Serial whenever the device is initialized 
https://github.com/adafruit/Adafruit_TinyUSB_Arduino/blob/master/src/arduino/Adafruit_USBD_Device.cpp#L220

main.cpp can explicitly call Serla.begin() afterwards without any issue if design ( if will just check a bool then return).